test/cli.spec.ts

```diff
@@ -0,0 +1,112 @@
+import sinon from 'sinon'
+import config from 'config'
+import path from 'path'
+import * as AppModule from '../src/app'
+import chai from 'chai'
+
+import RnsService from '../src/services/rns'
+import StorageService from '../src/services/storage'
+import { unlinkSync, copyFileSync, mkdirSync } from 'fs'
+import { sequelizeFactory } from '../src/sequelize'
+import { initStore } from '../src/store'
+import { getEndPromise } from 'sequelize-store'
+import StartCommand from '../src/cli/start'
+import { rmDir, sleep } from './utils'
+import sinonChai from 'sinon-chai'
+import * as blockchainUtils from '../src/blockchain'
+import { Eth } from 'web3-eth'
+import Rate from '../src/services/rates/rates.model'
+import { DbBackUpConfig } from '../src/definitions'
+
+chai.use(sinonChai)
+const expect = chai.expect
+
+describe('CLI', function () {
+  this.timeout(10000)
+  after(() => {
+    sinon.reset()
+    rmDir(config.get<DbBackUpConfig>('dbBackUp').path)
+  })
+  before(async () => {
+    // Prepare DB and set it to be used
+    const dbPath = path.join(__dirname, '..', '..', config.get<string>('db'))
+    try {
+      unlinkSync(dbPath)
+    } catch (e) {
+      // Ignore "not found" errors
+      if (e.code !== 'ENOENT') {
+        throw e
+      }
+    }
+    process.env.RIFS_DB = dbPath
```

Comment:
I see you copy&pasted a test from the Pinner. This won't work as the env. variable is `RIFM_DB` here. But tbh. not sure if it is needed? 🤔

Reply:
I think we can remove it